src/client/opamClient.ml

```diff
@@ -1158,9 +1158,12 @@ let remove_t ?ask ~autoremove ~force atoms t =
       OpamPackage.Set.of_list
         (OpamSolver.reverse_dependencies ~build:true ~post:true
            ~depopts:false ~installed:true universe packages) in
+    let base =
```

Comment:
Maybe adding a comment to explain why we don't take `universe.u_base` or `t.compilers_package`?